priv/templates/install/beacon_router_scope.ex

```diff
@@ -0,0 +1,14 @@
+
+  scope "/", BeaconWeb do
+    pipeline :beacon do
+      plug BeaconWeb.Plug
+    end
+
+    pipe_through :browser
+    pipe_through :beacon
+
+    live_session :beacon, session: %{"beacon_site" => "<%= beacon_site %>"} do
+      live "/beacon/*path", PageLive, :path
+    end
+  end
+end
```

Comment:
Would be better if we could remove this final `end` to keep just the `scope` in the template.